README.md

```diff
@@ -14,6 +14,22 @@ This SDK is powered by [TinyGo](https://tinygo.org/) and does not support the of
 - [TinyGo](https://tinygo.org/) - This SDK depends on TinyGo and leverages its [WASI](https://github.com/WebAssembly/WASI) (WebAssembly System Interface) target. Please follow the official instruction [here](https://tinygo.org/getting-started/) for installing TinyGo.
 - [Envoy](https://www.envoyproxy.io) - To run compiled examples, you need to have Envoy binary. We recommend using [func-e](https://func-e.io) as the easiest way to get started with Envoy. Alternatively, you can follow [the official instruction](https://www.envoyproxy.io/docs/envoy/latest/start/install).
 
+
+## Dealing with memory issues
+
+TinyGo's default memory allocator (Garbage Collector) is known to have some issues when it's used in the high workload environment (e.g. [1](https://github.com/tetratelabs/proxy-wasm-go-sdk/issues/349),[2](https://github.com/tetratelabs/proxy-wasm-go-sdk/issues/375)).
+There's an alternative GC called [nottinygc](https://github.com/wasilibs/nottinygc) which not only resolves the memory related issues, but
+also improves the performance on production usage.
+
+The following images are [an end user's observation](https://envoyproxy.slack.com/archives/CM2CQ4XJ5/p1682627227131899?thread_ts=1676594315.815989&cid=CM2CQ4XJ5) 
```

Comment:
Perhaps it's OK not to link to the thread as it requires an account so could be confusing if clicked